terraform/aks/config/development.tfvars.json

```diff
@@ -2,6 +2,15 @@
     "cluster": "test",
     "namespace": "git-development",
     "environment": "development",
+    "key_vault_resource_group":  "s189t01-gse-dv-rg",
+    "infra_key_vault_name": "s189t01-gse-dv-inf-kv",
     "azure_enable_backup_storage": false,
-    "enable_monitoring": false
+    "enable_monitoring": false,
+    "worker_apps": {
+        "worker": {
+          "startup_command": ["/bin/sh", "-c", "bundle exec sidekiq -C config/sidekiq.yml"],
```

Comment:
The startup command is always the same across all environments. Could we hardcode it in the terraform code to simplify?

Reply:
now hardcoded

Reply:
Please remove the line from the tfvars now